inputstream.rtmp/addon.xml.in

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <addon
   id="inputstream.rtmp"
-  version="2.0.8"
+  version="2.1.0"
```

Comment:
Can we change the version to 3.0.0?

I.e. Leia is version v2.X.X, Matrix is v3.X.X. This way its much clearer for users and everyone else.

Reply:
NP, I change :smile: